packages/rum-core/src/performance-monitoring/transaction.js

```diff
@@ -91,11 +91,10 @@ class Transaction extends SpanBase {
     if (this.ended) {
       return
     }
-    var transaction = this
-    var opts = extend({}, options)
+    const opts = extend({}, options)
 
-    opts.onEnd = function (trc) {
-      transaction._onSpanEnd(trc)
+    opts.onEnd = trc => {
```

Comment:
I think it's better to keep this a function since we're assigning it on the options. It's not going to be a problem with our current code, but I think it's better to be explicit.

Reply:
We transpile it anyway which gets converted to Es5. I did this because some of the code were Es6 and rest were Es5. Its better to move everything to ES6 and bundle accordingly